controller/target/classes/application.properties

```diff
@@ -0,0 +1 @@
+server.port = 8089
```

Comment:
Remove the /target or add them to .gitIgnore. Compiled files shouldn't be part of the repository code